### ISSUE TITLE
Broken: (Make Bone2D Nodes from Nodes)

### ISSUE THREAD

**Issue by xiaoandtell** [godotengine/godot#92227]
### Tested versions

Reproducible in v4.2.2.stable.official [15073afe3]
Not Reproducible in v3.5.3.stable

### System information

Windows 10 + v4.2.2.stable.official 

### Issue description

Selecting a chain of sprites nodes and clicking Make Bone2d Nodes from Nodes results in each sprite node being reparented to disjointed bone nodes with warning: 
- A Bone2D only works with... as parent node
- This bone lacks a REST pose
![bug](https://github.com/godotengine/godot/assets/1452746/c321dab3-90de-4701-b3b4-a43152157aab)

### Steps to reproduce

1. Create a parented chain of sprite nodes
-Upperarm
--Forearm
---Hand
2. Move them to the proper pivot points
3. Select the 3 nodes and hit Ctrl+Shift+B (Make Bone2d Nodes from Nodes  under the bone button)

### Minimal reproduction project (MRP)

[Godot2dSkelV4Bug.zip](https://github.com/godotengine/godot/files/15395923/Godot2dSkelV4Bug.zip)

**Comment by jinguoliang** [comment]
I learned cutout animation by the doc: https://docs.godotengine.org/en/4.2/tutorials/animation/cutout_animation.html
Maybe it is outdate，or the behavor of editor is not keep with the doc

**Comment by CsloudX** [comment]
For developers who follow the tutorial  https://docs.godotengine.org/en/4.2/tutorials/animation/cutout_animation.html to learn Godot, this issue experience is very unfriendly. So, can we prioritize fixing this bug?

**Comment by Tsaot** [comment]
I've been able to recreate this issue using 4.4.1. Also using the same cutout animation tutorial as an example.

**Comment by F3der1co** [comment]
yes I still encounter this in 4.5.beta1. The behavior is broken, The bones are created incorrectly:

Observed:

<img width="284" alt="Image" src="https://github.com/user-attachments/assets/8e927b8d-ca5b-4db1-9a02-8aabe77ceadf" />

Expected:

<img width="270" alt="Image" src="https://github.com/user-attachments/assets/ecfc6eff-1e81-42e9-8453-cb4c5558c440" />

**Comment by Joey-Einerhand** [comment]
This has been broken since the new bone system in 4.0

**Comment by blam23** [comment]
Any update on this? It's a pretty big barrier to entry for the bone system (neither a fix for the tutorial nor the PR to fix this has been merged)

**Comment by Joey-Einerhand** [comment]
> Any update on this? It's a pretty big barrier to entry for the bone system (neither a fix for the tutorial nor the PR to fix this has been merged)

It's not ideal, but the[ 2D skeleton tutorial](https://docs.godotengine.org/en/stable/tutorials/animation/2d_skeletons.html) covers how to use bone2d/skeleton2d without this button.
